tutorials/scripting/cross_language_scripting.rst

```diff
@@ -173,25 +173,22 @@ If that's impossible, you'll see the following error: ``Invalid call. Nonexisten
 
 Receiving C# collections from GDScript
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-When passing or returning collections from C# methods, types from the `System.Collections` and `System.Collections.Generic` namespaces are not compatible with Godot's marshalling system. For example:
+When passing or returning collections from C# methods, types from the `System.Collections` and `System.Collections.Generic` namespaces are not compatible with Godot's marshalling system. For example, this method will not be visible to GDScript, and Godot will throw the "Invalid call" error:
```

Comment:
```suggestion
When passing or returning collections from C# methods, types from the `System.Collections` and `System.Collections.Generic` namespaces are not compatible with Godot's marshalling system. For example, this method will not be visible to GDScript, and Godot will throw an ``Invalid call`` error:
```

Style nitpick to match the usage of `code` for errors in the rest of the page; and use "an" to not assume the reader has read the whole page. Other than that, LGTM!